qml/ActionsToolbar.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.9
```

Comment:
Something new and interesting... When using this under a right-to-left language, the cursor moves opposite to your finger. I don't know how to write or read a right-to-left language, but I suspect this is not what they'd want.

Reply:
I actually noticed this too but I haven't tried to find a fix yet since it's an existing bug. Let me check in the C++ part if we can find something we can check to invert the horizontal directions.

Reply:
Sadly, I wasn't able to find anything that can help with this from the C++ part and QML part. To be fair, the same behavior happens in Notepad++ on Windows 10 when setting to RTL 😀 
Perhaps this is something we can just create a separate issue?

Reply:
Interesting. Sometimes it works, sometimes it doesn't. The cursor movement seems broken in general with RTL languages... but only with on-screen keyboard. An attached hardware keyboard can move the cursor normally.

Reply:
I've tried different scenarios using Arabic. All in edge channel though.
The same behavior happened when I used the OSK and a bluetooth KB in Jotit app.
I tried again in QML Creator and the same thing happened - cursor movements are reversed.
But then I tried in messaging app, same thing happened until I tapped on the space bar and the text went from left to right. I figured this happened because I have word suggestions enabled.
So I experimented further, and it seems that text fields can only be LTR or RTL and I guess it cannot be changed anymore. I had a scenario when I typed Arabic first so the text are on the right then I typed English texts and the cursor movement are now reversed in the English texts.

So my conclusion is that the behavior depends on the direction of the text field (RTL/LTR) and the direction of the actual texts. If the field is in LTR, you'll get reverse cursor movement on RTL languages and when the field is RTL, the LTR languages will get the reverse movement.
With that, I guess it'll be not much of a problem for RTL users since they most likely start with RTL in text fields.

That was a long post and hopefully helpful 😀